site/src/styl/index.styl

```diff
@@ -101,3 +101,6 @@ footer
         width 100%
 
 @import "imports/a11y"
+
+#content .item.visible
```

Comment:
I know there's probably a better way of doing this; if you have styl experience please suggest changes 😄